### PR TITLE
Fix: Frontend spacing issues

### PR DESCRIPTION
**Pull request by vijaivir**
# Description
Fixes the spacing for some alerts and text on the frontend. 

https://bcdevex.atlassian.net/browse/FORMS-1147
https://bcdevex.atlassian.net/browse/FORMS-1035

### Before

image-20240306-163428
image-20240509-232529
image-20240418-212831

### After

Screenshot 2024-08-13 at 4 42 30 PM
Screenshot 2024-08-13 at 4 46 56 PM
Screenshot 2024-08-13 at 4 44 52 PM

## Types of changes

fix (a bug fix)

## Checklist

- [x] I have read the [CONTRIBUTING](/bcgov/common-hosted-form-service/blob/main/CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have run the npm script lint on the frontend and backend
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] I have approval from the product owner for the contribution in this pull request

### CODE REVIEW

File: app/frontend/src/components/designer/settings/FormScheduleSettings.vue [hunk 1]
```diff
@@ -577,11 +577,13 @@ defineExpose({
...
 
-        <span :lang="locale">{{
+        <span :lang="locale" data-test="late-submission-text">{{
           form.schedule.allowLateSubmissions.enabled &&
           form.schedule.allowLateSubmissions.forNext.intervalType &&
           form.schedule.allowLateSubmissions.forNext.term
-            ? $t('trans.formSettings.allowLateSubmissnInterval') +
+            ? ' ' +
+              $t('trans.formSettings.allowLateSubmissnInterval') +
+              ' ' +
```
Comment (WalterMoar) [inline app/frontend/src/components/designer/settings/FormScheduleSettings.vue:586]: Hmm, I think this is adding an extra space in this case:
image
Looks like there's also another missing space around line 605 for `1months`?

File: app/frontend/src/components/forms/submission/ManageSubmissionUsers.vue [hunk 2]
```diff
@@ -174,8 +174,10 @@ async function modifyPermissions(userId, permissions) {
         ...NotificationTypes.SUCCESS,
         text: permissions.length
           ? t('trans.manageSubmissionUsers.sentInviteEmailTo') +
+            ' ' +
             `${selectedEmail}`
           : t('trans.manageSubmissionUsers.sentUninvitedEmailTo') +
+            ' ' +
```
Comment (WalterMoar) [inline app/frontend/src/components/forms/submission/ManageSubmissionUsers.vue:180]: Looks great! :)

**Review by WalterMoar — DISMISSED** [review]
The work for FORMS-1147 looks great, and love that it has tests! For FORMS-1035 I think it still needs a bit of tweaking to handle all the cases.

File: app/frontend/src/components/designer/settings/FormScheduleSettings.vue (hunk 1 above)
Comment (vijaivir) [reply app/frontend/src/components/designer/settings/FormScheduleSettings.vue:584]: Good catch! Its fixed now. 

Screenshot 2024-08-19 at 11 35 49 AM

**Review by WalterMoar — COMMENTED** [review]
Functionally this looks great to me! Everything is working as expected. I'll let @jasonchung1871 give the final thumbs up on code style, etc, as he's much more familiar with it than I am.

**Comment by vijaivir** [comment]
> Functionally this looks great to me! Everything is working as expected. I'll let @jasonchung1871 give the final thumbs up on code style, etc, as he's much more familiar with it than I am.

Sounds good. Thanks for checking it out!

**Review by jasonchung1871 — APPROVED** [review]

**Pull request merged**